docs/installer/lokoctl.md

```diff
@@ -20,26 +20,26 @@ These binaries can be manually downloaded and installed.
    keys](https://github.com/kinvolk/lokomotive/blob/master/docs/KEYS.md).
```

Comment:
This commit has an empty `scripts/update/0.6.1-0.7.0/update.sh` file

Reply:
Fixed.